es-modules/src/components/todo-app.js

```diff
@@ -0,0 +1,58 @@
+// const { createElement, noop } = require('../lib');
+import { createElement, noop } from '../lib';
+// const TodoList = require('./todo-list');
+import TodoList from './todo-list';
+// const TodoForm = require('./todo-form');
+import TodoForm from './todo-form';
```

Comment:
Please delete all of the commented out lines.

Reply:
it's been deleted!